recipes/odbc/all/conanfile.py

```diff
@@ -65,9 +71,17 @@ def _configure_autotools(self):
         self._autotools.configure(configure_dir=self._source_subfolder, args=args)
         return self._autotools
 
+    @property
+    def _user_info_build(self):
+        return getattr(self, "user_info_build", None) or self.deps_user_info
```

Comment:
```suggestion
        return getattr(self, "user_info_build", self.deps_user_info)
```

Reply:
Could also be used for _settings_build in others recipe:

```python
    @property
    def _settings_build(self):
        return getattr(self, "settings_build", self.settings)
```